config/webpack.plugins.js

```diff
@@ -86,6 +87,18 @@ const plugins = (dev = false, beta = false, restricted = false) => {
       __SENTRY_DEBUG__: false,
     }),
     ...(dev ? [new ReactRefreshWebpackPlugin()] : []),
+    // Put the Sentry Webpack plugin after all other plugins
+    sentryWebpackPlugin({
+      authToken: process.env.SENTRY_AUTH_TOKEN,
```

Comment:
What if the token does not exist?

Reply:
I will create a secret variable for this, so itll essentially always be there when we merge, but if a user doesnt want to build it locally every time then it wont build

Reply:
Can we just not include the plugin if the env variable does not exist? And gate it behind the `SENTRY_DSN` as well?